fs/config/config.go

```diff
@@ -101,7 +101,7 @@ type FuseConfig struct {
 	NegativeTimeout int64 `toml:"negative_timeout"`
 
 	// LogFuseOperations enables logging of operations on FUSE FS. This is to be used
-	// for debugging purposes only.
+	// for debugging purposes only. This option may emit sensitive information.
```

Comment:
nit: might be good to provide some examples of what types of sensitive information may be emitted.

Reply:
Yeah I think we should explicitly the fact that this will enable emitting filepaths/names within the container.